rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+ignore = [
+    "lib/serviceinfo/models",
+    "lib/tes/models",
+]
```

Comment:
Is there a way to specify patterns? In that case you could add sth like `lib/*/models` instead of listing the individual files.